homeassistant/components/huawei_lte/sensor.py

```diff
@@ -173,6 +180,23 @@ class SensorMeta(NamedTuple):
             "mdi:signal-cellular-3",
         )[bisect((-20, -10, -6), x if x is not None else -1000)],
     ),
+    (KEY_DEVICE_SIGNAL, "transmode"): SensorMeta(name="Transmission mode"),
+    (KEY_DEVICE_SIGNAL, "cqi0"): SensorMeta(
+        name="CQI 0",
+        icon="mdi:speedometer",
+    ),
+    (KEY_DEVICE_SIGNAL, "cqi1"): SensorMeta(
+        name="CQI 1",
+        icon="mdi:speedometer",
+    ),
+    (KEY_DEVICE_SIGNAL, "ltedlfreq"): SensorMeta(
+        name="Downlink frequency",
+        formatter=lambda x: (int(x) / 10.0, "MHz"),
```

Comment:
Do we want to make this always a float, would int not be appropriate? I.e. divide by just 10. What kind of values do you see for this and the uplink one?

Reply:
I'm fine with rounding. See the updated code.